packages/client-config/src/client-config-contributor/client_config_contributor_v1.test.ts

```diff
@@ -540,6 +540,11 @@ void describe('storage client config contributor v1', () => {
         name: 'testName',
         bucketName: 'testBucketName',
         storageRegion: 'testRegion',
+        paths: {
+          'path/*': {
+            guest: ['get'],
+          },
+        },
```

Comment:
Can you add all five types of access rules we support here.

For groups, entity and resource, do we print the actual substituted value (for the group, entity and resource) or just the key name `groups`, `entity` etc.

Reply:
Maybe doesn't add much value in this test, since we are just mapping payload to client config. I'll add more to `storage_access_orchestrator.test.ts`

Reply:
Is this updated? I was thinking this is where we should update https://github.com/aws-amplify/amplify-backend/pull/1927/files#diff-cc440dd1098daee47d41294de3ae883a0c8e2570e516061b54623fa84f7ff8edR136-R139

Reply:
I added all the access rules but we won't see much benefit from adding more to this test since we are just mapping paths from the payload to the config. For key names, this is handled in `access_builder.test.ts` and tested there and for verifying the `paths` is built correctly that is done when we assert `storageAccessDefinitionOutput` in `storage_access_orchestrator.test.ts`.